hawc/apps/riskofbias/exports.py

```diff
@@ -23,17 +24,16 @@ def _get_data_rows(self):
             row.extend(Study.flat_complete_data_row(ser))
 
             try:
-                scores = [
```

Comment:
this could be problematic to maintain if we add new fields. Could we use `common.helper.df_move_column` instead?